### PR TITLE
fix: Add missing directory to NGINX App Protect DoS Instructions

### PR DESCRIPTION
**Pull request by ADubhlaoich**
This miniature commit changes the path for the CD command in the second prerequisite of the "Installation with NGINX App Protect DoS" document. A reader with existing knowledge would spot the issue immediately, but one without might be left confused why the subsequent commands fail.

Additional, this commit fixes some minor noun issues for the documentation based on the product naming conventions.

### Proposed changes
Describe the use case and detail of the change. If this PR addresses an issue on GitHub, make sure to include a link to that issue here in this description (not in the title of the PR).

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [x] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by jjngx — APPROVED** [review]

**Review by jjngx — APPROVED** [review]

**Review by vepatel — APPROVED** [review]

**Review by haywoodsh — APPROVED** [review]

File: docs/content/app-protect-dos/installation.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+title: Installation with NGINX App Protect DoS
+description: "This document provides an overview of the steps required to use NGINX App Protect DoS with your NGINX Ingress Controller deployment."
 weight: 1800
 doctypes: [""]
 toc: true
 docs: "DOCS-583"
 ---
 
-> **Note**: The NGINX Kubernetes Ingress Controller integration with NGINX App Protect DoS requires the use of NGINX Plus.
+> **Note**: The F5 NGINX Kubernetes Ingress Controller integration with NGINX App Protect DoS requires the use of NGINX Plus.
```
Comment (lucacome) [inline docs/content/app-protect-dos/installation.md:10]: doesn't it have to be `F5 NGINX App Protect DoS` and `F5 NGINX Plus` then?

Comment (brianehlert) [reply docs/content/app-protect-dos/installation.md:10]: The very first mention of the product name on the page requires the "F5" prepend.
This is not required in Notes within the body of the page.  This is not required for every mention of the product names.
The official product names remain "NGINX Ingress Controller" (without 'Kubernetes'), "NGINX Plus", and "NGINX App Protect DoS"

Comment (ADubhlaoich) [reply docs/content/app-protect-dos/installation.md:10]: @brianehlert I think @lucacome was wondering if this convention applied to the first mention of any product name, as opposed to the product that a given document is written/centered around. I'm not actually sure about this, so I'll check in with my team.

Comment (ADubhlaoich) [reply docs/content/app-protect-dos/installation.md:10]: Having checked in with my team, it is in fact the first mention of any product name in any document. Pushing an update.

**Review by lucacome — APPROVED** [review]

**Pull request merged**